governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx

```diff
@@ -230,7 +255,40 @@ const Proposal = ({
 }) => {
   const [instructions, setInstructions] = useState<MultisigInstruction[]>([])
   const [isTransactionLoading, setIsTransactionLoading] = useState(false)
-  const { cluster } = useContext(ClusterContext)
+  const { cluster: contextCluster } = useContext(ClusterContext)
+  const multisigCluster = getMultisigCluster(contextCluster)
+  const targetClusters: (PythCluster | 'unknown')[] = []
+  instructions.map((ix) => {
+    if (ix instanceof PythMultisigInstruction) {
+      targetClusters.push(multisigCluster)
+    } else if (
+      ix instanceof WormholeMultisigInstruction &&
+      ix.governanceAction instanceof ExecutePostedVaa
+    ) {
+      ix.governanceAction.instructions.map((ix) => {
+        const remoteClusters: PythCluster[] = [
+          'pythnet',
+          'pythtest-conformance',
+          'pythtest-crosschain',
+        ]
+        for (const remoteCluster of remoteClusters) {
+          if (
+            multisigCluster === getMultisigCluster(remoteCluster) &&
+            ix.programId.equals(getPythProgramKeyForCluster(remoteCluster))
+          ) {
+            targetClusters.push(remoteCluster)
+          }
+        }
+      })
+    } else {
+      targetClusters.push('unknown')
+    }
+  })
+  const uniqueTargetCluster = new Set(targetClusters).size === 1
```

Comment:
Maybe show a warning on this case?

Reply:
We do. I show a `No target Pyth program detected` or a `Multiple target Pyth programs detected` message

Reply:
You mean in terms of icon/color?

Reply:
aha right sorry i missed it. ok ok.